frontend/js/require.config.js

```diff
@@ -27,6 +27,7 @@ require.config({
         "i18next-browser-language-detector": "libs/i18nextBrowserLanguageDetector.min",
         "jquery-i18next": "libs/jquery-i18next.min",
         "moment": "libs/moment-with-locales",
+        "Hls":"libs/mediaelement/renderers/hls",
```

Comment:
There should be a space after the colon. :)

Reply:
Done!